### PR TITLE
Add university

### PR DESCRIPTION
**Pull request by alyssaongyx**

### CODE REVIEW

File: src/main/java/seedu/address/model/university/UniqueUniversityList.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    private final ObservableList<University> internalList = FXCollections.observableArrayList();
+    private final ObservableList<University> internalUnmodifiableList =
+            FXCollections.unmodifiableObservableList(internalList);
+
+    public boolean contains(University toCheck) {
+        requireNonNull(toCheck);
+        return internalList.stream().anyMatch(toCheck::isSameUniversity);
+    }
+
+//    public void add(University toAdd) {
```
Comment (aliciamichellew) [inline src/main/java/seedu/address/model/university/UniqueUniversityList.java:27]: Can I ask why is this commented out? I think if it is not needed, we can remove it

Comment (alyssaongyx) [reply src/main/java/seedu/address/model/university/UniqueUniversityList.java:27]: i think we are not having the command for adding of universities

Comment (aliciamichellew) [reply src/main/java/seedu/address/model/university/UniqueUniversityList.java]: Can we remove it?

Comment (alyssaongyx) [reply src/main/java/seedu/address/model/university/UniqueUniversityList.java]: yep ive removed it already, pushed the new changes

**Review by aliciamichellew — APPROVED** [review]
Thank you! LGTM!

**Review by dlathyun — APPROVED** [review]
LGTM!

**Pull request merged**